pkg/config/types/v1alpha1/v1alpha1_configurator.go

```diff
@@ -0,0 +1,348 @@
+// This Source Code Form is subject to the terms of the Mozilla Public
```

Comment:
This is just a lift of all the methods that were split across multiple files and dropping them into a single file.